### ISSUE TITLE
[Feature Request] Implement metrics for relay service

### ISSUE THREAD

**Issue by cromefire** [syncthing/syncthing#5116]
Implement the metrics for the relay service, like it is in the discovery server.

### Version Information

Syncthing Version: `v0.14.49`
OS Version: `Ubuntu 18.04 LTS`
Browser Version: not applicable

**Comment by calmh** [comment]
The relay service exposes metrics on the status port/service, which is what relays.syncthing.net is built from.

**Comment by cromefire** [comment]
But they are not Prometheus compatible like `/metrics` in the discovery server.

**Comment by cromefire** [comment]
sure, they can use the same code, but shoul output a timeseries that I can use to determine my load / error from my grafana dashboard

**Comment by calmh** [comment]
Yeah, this code predates Prometheus, or at least widespread usage of it. I’m going to say “maybe, low prio, PR probably accepted”.

**Comment by cromefire** [comment]
I don't know go and it's universe so I can't really and do a PR

**Comment by AudriusButkevicius** [comment]
You could scrape https://relays.syncthing.net/metrics and filter out your own relay in the meanwhile.

**Comment by cromefire** [comment]
It's a private relay (for various reasons I can't make it public, but I don't use public ones for Gigabytes of data), but I can live without it. I already have a Prometheus for my other stuff, so I'd love to also see metrics of my relay.

**Comment by AudriusButkevicius** [comment]
Sure. But as I said abive, nobody cares about this as we track oublic relays via the pool, so I wouldn't hold your breath for this happening.

**Comment by lukasmrtvy** [comment]
I guess that it should not be hard to create prometheus exporter binary for scraping stats endpoint just like strelaypoolsrv ( https://github.com/syncthing/syncthing/blob/master/cmd/strelaypoolsrv/stats.go#L29 ) does it.
